cmd/clusterawsadm/cloudformation/bootstrap/cluster_api_controller.go

```diff
@@ -147,6 +147,7 @@ func (t Template) ControllersPolicy() *iamv1.PolicyDocument {
 				"elasticloadbalancing:DeleteTargetGroup",
 				"elasticloadbalancing:DescribeLoadBalancers",
 				"elasticloadbalancing:DescribeLoadBalancerAttributes",
+				"elasticloadbalancing:DescribeTargetGroups",
```

Comment:
elasticloadbalancing:DescribeTargetGroups action is needed for capa when AlternativeGCStrategy enabled.
Should I just enable it in the capa code as a basic action? Or should I custom it when running e2e test when rendering cloudformation just like https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/main/test/e2e/shared/template.go#L150-L153?

Reply:
As we create the CF stack with every e2e run this will be picked up automatically.

Reply:
We need to flag this as **action required** in the release notes section so that users know they need to update their CF stack if they want to use the new alternative strategy.

Reply:
Thanks Richard.

How about  I change the issue tile "E2E tests for AlternativeGCStrategy" to "Add required permission to CloudFormation to support AlternativeGCStrategy"
and add release notes block in this issue:
**action required**: user need to update CF stack to use AlternativeGCStrategy.

Am I understanding right.